app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
Using a class variable 'alive' to track instances can lead to unintended side effects, especially in a multi-threaded environment. Consider using instance variables or another mechanism to track living animals.